libs/designer/src/lib/ui/panel/panelRoot.tsx

```diff
@@ -70,14 +99,39 @@ export const PanelRoot = (props: PanelRootProps): JSX.Element => {
     </div>
   );
 
+  const resize = useCallback(
+    ({ clientX }: MouseEvent) => {
+      animationFrame.current = requestAnimationFrame(() => {
+        if (isResizing) {
+          const newWidth = window.innerWidth - clientX < 300 ? 300 : window.innerWidth - clientX;
```

Comment:
Nit: Can use the `Math.max()` function here instead of the  conditional
ie. `Math.max(window.innerWidth - clientX, 300)`

Reply:
Sure thing, thanks!